corehq/ex-submodules/casexml/apps/phone/tests/test_sync_mode.py

```diff
@@ -165,7 +156,23 @@ def _testUpdate(self, sync_log_or_id, case_id_map, dependent_case_id_map=None):
             self._testUpdate(migrated_sync_log, case_id_map, dependent_case_id_map)
 
 
-class SyncTokenUpdateTest(SyncBaseTest):
+class SyncBaseTest(BaseSyncTest):
```

Comment:
Good idea. I'll do it in a separate PR. It's used in a lot of places, which is why I did not change it in this PR.